lib/features/login/ui/login_screen.dart

```diff
@@ -8,7 +8,7 @@ class LoginScreen extends StatelessWidget {
   @override
   Widget build(BuildContext context) {
     return Scaffold(
-      appBar: loginAppBar(),
+      appBar: authMethodsAppBar('Welcome Back!'),
```

Comment:
please rename it to AuthMethodsAppBar

Reply:
But this is a function not a widget

Reply:
then it's better to start with a verb, buildAuthMethodsAppBar or setupAuthMethodsAppBar